sourced/ml/__main__.py

```diff
@@ -35,6 +35,45 @@ def one_arg_parser(*args, **kwargs) -> argparse.ArgumentParser:
     return arg_parser
 
 
+def add_spark_args(my_parser):
+    my_parser.add_argument(
+        "-s", "--spark", default=SparkDefault.MASTER_ADDRESS,
+        help="Spark's master address.")
+    my_parser.add_argument(
+        "--config", nargs="+", default=SparkDefault.CONFIG,
+        help="Spark configuration (key=value).")
+    my_parser.add_argument(
+        "-m", "--memory",
+        help="Handy memory config for spark. -m 4,10,2 is equivalent to "
+             "--config spark.executor.memory=4G "
+             "--config spark.driver.memory=10G "
+             "--config spark.driver.maxResultSize=2G."
+             "Numbers are floats separated by commas.")
+    my_parser.add_argument(
+        "--package", nargs="+", default=SparkDefault.PACKAGE,
+        help="Additional Spark package.")
+    my_parser.add_argument(
+        "--spark-local-dir", default=SparkDefault.LOCAL_DIR,
+        help="Spark local directory.")
+    persistences = ("DISK_ONLY", "DISK_ONLY_2", "MEMORY_ONLY", "MEMORY_ONLY_2",
```

Comment:
it's better to use:
```python
In[10]: [att for att in pyspark.StorageLevel.__dict__.keys() if "__" not in att]
Out[10]: 
['MEMORY_ONLY_SER_2',
 'MEMORY_AND_DISK_2',
 'DISK_ONLY',
 'MEMORY_ONLY',
 'OFF_HEAP',
 'MEMORY_AND_DISK',
 'MEMORY_AND_DISK_SER_2',
 'MEMORY_AND_DISK_SER',
 'MEMORY_ONLY_2',
 'DISK_ONLY_2',
 'MEMORY_ONLY_SER']
```
because there are more options (11) than you have in your list

Reply:
Agree, let's postpone it to an issue and fix later